### PR TITLE
Add network to clu

### PR DESCRIPTION
**Pull request by DeepakRajendranMsft**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@DeepakRajendranMsft__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

File: src/CLU/CLUCoreCLR.sln [hunk 1]
```diff
@@ -64,96 +64,190 @@ Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Sync", "Sync\Sync.xproj", "
...
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug|Any CPU = Debug|Any CPU
+		Debug|x64 = Debug|x64
 		Release|Any CPU = Release|Any CPU
+		Release|x64 = Release|x64
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
 		{5F567ACA-595E-436D-83DB-A21E08F82DF6}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{5F567ACA-595E-436D-83DB-A21E08F82DF6}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{5F567ACA-595E-436D-83DB-A21E08F82DF6}.Debug|x64.ActiveCfg = Debug|Any CPU
```
Comment (markcowl) [inline src/CLU/CLUCoreCLR.sln]: Why all the x64 builds?

Comment (DeepakRajendranMsft) [inline src/CLU/CLUCoreCLR.sln]: will remove that.

**Comment by markcowl** [comment]
@DeepakRajendranMsft Please remove Nuget.Config, which refers to a path on your system.

**Comment by markcowl** [comment]
@DeepakRajendranMsft Please remove update to global.json - this will require changes across the board in dependencies

**Comment by markcowl** [comment]
@DeepakRajendranMsft fix spacing in BuildCmdlet.bat

**Comment by markcowl** [comment]
@DeepakRajendranMsft lgtm once the build passes

**Pull request merged**